config/default.yml

```
@@ -4329,10 +4329,13 @@ Style/RegexpLiteral:
   # slashes: Always use slashes.
   # percent_r: Always use `%r`.
   # mixed: Use slashes on single-line regexes, and `%r` on multi-line regexes.
+  # mixed_preserve: Preserve current style and only use `%r` if AllowInnerSlashes=true
```

Should it be `AllowInnerSlashes=false` instead? 🤔 

```suggestion
  # mixed_preserve: Preserve current style and only use `%r` if AllowInnerSlashes=false
```